src/components/CardList.tsx

```diff
@@ -1,19 +1,41 @@
 import Card, {CardProps} from "./Card";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import {theme} from "../utils/theme";
 
 const CardList: React.FC<{cards: CardProps[]}> = ({cards}) => {
     return (
-        <div className="card-list">
-            <h2>Stadium list</h2>
-            {cards.length > 0 ? (
-                <>
-                    {cards.map((card, index) => (
-                        <Card key={index} {...card} />
-                    ))}
-                </>
-            ) : (
-                <p>No cards available</p>
-            )}
-        </div>
+        <Box className="card-list">
+            <Typography variant="h4">Stadium list</Typography>
+            <Container>
+                {cards.length > 0 ? (
+                    <Grid
+                        container
+                        spacing={1}
+                        columns={{xs: 1, md: 2, lg: 2}}
+                        sx={{
+                            overflowY: {sm: "auto"},
+                            maxHeight: "800px",
+                            "&::-webkit-scrollbar": {width: "0.4em"},
+                            "&::-webkit-scrollbar-thumb": {
+                                backgroundColor: theme.palette.primary.dark,
+                                borderRadius: "10px",
```

Comment:
borderRadius can also be taken from theme. For this you need to use shape param. 
First you need to extend it
```
 interface LibraryShape {

  border: {
    xs: number;
    sm: number;
  };
}
``` 
in your theme file and then use like a pallet

 ```
shape: {
    border: {
      xs: 1,
      sm: 2
    }
  },
```
I provided you example with border, it works the same for borderRadius
If there is any questions feel free to ask :)